docs/index.html

```diff
@@ -134,13 +134,20 @@
 
             // Used from https://github.com/ckoliber/docsify-rtl/blob/master/build/docsify-rtl.js,
             // Currently this is a hack because can't use the plugin as is.
-            if (vm.route.path.search('Arabic') !== -1) {
+            if (vm.route.path.search('Arabic') !== -1 || vm.route.path.search('Hebrew') !== -1) {
```

Comment:
Added Hebrew here as I believe the language should also be read from right to left